app/server/models/credit_transfer.py

```diff
@@ -63,6 +59,15 @@ class CreditTransfer(ManyOrgBase, BlockchainTaskableBase):
     to_exchange = db.relationship('Exchange', backref='to_transfer', lazy=True, uselist=False,
                                   foreign_keys='Exchange.to_transfer_id')
 
+    # TODO: Apply this to all transfer amounts/balances, work out the correct denominator size
+    @hybrid_property
+    def transfer_amount(self):
+        return self._transfer_amount / 10**6
```

Comment:
not a fan of this naming. would prefer it being more precise so we know what we're working with. `transfer_amount_cents` instead of just `_transfer_amount` for example, but whatever the terminology is with crypto

Reply:
yeah good point. How about we
- Go to storing full 18 decimal precision data using the `numeric` datatype
- call this `transfer_amount_wei`, because that's unambiguously 1 * 10^-18 USD/ETH

When passing around amounts use one of
a) Wei (will low power android devices die if they have to store 18+ sig figs?)
b) cents
c) float

@wangfowen @tristanhcole

Reply:
going to swap the amounts stored on the db to full precision right now because that's a relatively small change, and we can work out to do with the other locations later because there's implications for data transfer etc.